auth/src/main/java/com/firebase/ui/auth/util/SmartlockUtil.java

```diff
@@ -12,44 +12,54 @@
  * Helper class to deal with Smartlock Flows.
  */
 public class SmartlockUtil {
-
     /**
      * If SmartLock is enabled and Google Play Services is available, start the save credential
      * Activity. Otherwise, finish the calling Activity with RESULT_OK.
-     * @param activity the calling Activity.
-     * @param requestCode request code to use when starting the save operation.
-     * @param parameters calling Activity flow parameters.
+     *
+     * @param activity     the calling Activity.
+     * @param requestCode  request code to use when starting the save operation.
+     * @param data         An Intent to be merged. Must be a {@code new Intent().putExtras(...)}.
```

Comment:
YESSSSS!!!! It took me like a half hour of debugging, but it finally dawned on me to actually look at what the test was doing:

Real life in `AcquireEmailHelper`:
```java
if (email != null && !email.isEmpty()) {
    // Only proceeds if email is not null
}
```

Test:
```java
Intent startIntent = SignInActivity.createIntent(
        RuntimeEnvironment.application,
        TestHelper.getFlowParameters(
                RuntimeEnvironment.application,
                Collections.<String>emptyList()),
        ***null***);
```

So anyway, I fixed the test and life is good again. The end.